resources/definitions/z-bolt_base.def.json

```diff
@@ -0,0 +1,199 @@
+		"layer_height_0":				{ "value": "layer_height if layer_height >= 0.24 else 0.2"},
+		"line_width":                                   { "value": "machine_nozzle_size" },
+		"wall_line_width":                              { "value": "machine_nozzle_size * 1.2" },
+		"infill_line_width":                            { "value": "machine_nozzle_size * 1.1" },
+		"wall_thickness":				{ "value": "wall_line_width * 3"},
+		"optimize_wall_printing_order":                 { "default_value": true },
+		"travel_compensate_overlapping_walls_enabled":  { "default_value": true },
+		"fill_perimeter_gaps":                          { "default_value": "everywhere" },
+		"filter_out_tiny_gaps": 			{ "value": true },
+		"fill_outline_gaps":                            { "default_value": false },
+		"jerk_enabled":                                 { "default_value": false },
+		"acceleration_enabled":                         { "default_value": false },
+		"zig_zaggify_infill":                           { "value": true },
+		"infill_before_walls":                          { "default_value": false },
+		"fill_outline_gaps":                            { "default_value": true },
+		"meshfix_maximum_resolution":                   { "default_value": 0.01 },
+		"z_seam_type":                       		{ "value": "'sharpest_corner'" },
+		"z_seam_corner":				{ "value": "'z_seam_corner_outer'" },
+		"roofing_layer_count":				{ "value": 1 },
+		"top_bottom_thickness":				{ "value": "layer_height * 5" },
+		"top_bottom_pattern":                		{ "value": "'zigzag'" },
+		"top_bottom_pattern_0":              		{ "value": "'zigzag'" },
+		"skin_overlap":					{ "value": 25.0 },
+		"infill_pattern":                    		{ "value": "'cubic'" },
+		"zig_zaggify_infill":                		{ "value": true },
+		"infill_overlap": 				{ "value": 25.0 },
+		"infill_before_walls": 				{ "value": false },
+		"material_initial_print_temperature":		{ "value": "material_print_temperature" },
+		"material_final_print_temperature":		{ "value": "material_print_temperature" },
+		"speed_infill":					{ "value": "speed_print" },
+		"speed_wall":                                   { "value": "math.ceil(speed_print * 0.6)" },
+		"speed_wall_0":                                 { "value": "math.ceil(speed_print * 0.6)" },
+		"speed_wall_x":                                 { "value": "math.ceil(speed_print * 0.8)" },
+		"speed_roofing":                                { "value": "math.ceil(speed_print * 0.4)" },
+		"speed_topbottom":                              { "value": "math.ceil(speed_print * 0.5)" },
+		"speed_prime_tower": 				{ "value": "speed_topbottom" },
+		"speed_support": 				{ "value": "speed_print" },
+		"speed_support_interface": 			{ "value": "speed_topbottom" },
+		"speed_travel":                                 { "value": 120, "maximum_value": 200 },
```

Comment:
No, we want our customers to use firmware built-in feedrates and acceleration values.

Reply:
This has nothing to do with the acceleration values.

If you want users to use the feedrate defined in the firmware, you'd need to set all speeds to 0 which would prevent Cura from outputting any `F` parameters with its movements. The current proposal would be wrong then as well, because this defines the feedrate to be 120mm/s (F7200), not to be firmware-defined but Cura-defined.

Either way, if the firmware imposes a limit on feedrate of 200mm/s in the X direction, setting `machine_max_feedrate_x` to 200 would limit the user in the same way as with the `speed_travel` set to 200, but offers more of an explanation as to why the feedrate is higher than possible. And it would also properly limit the other speeds, such as the infill speed. Important to note though is that the X and Y feedrates are normally limited separately by firmware, so the printer could still move diagonally at 282.8mm/s. If this limit works differently on your printer, you can override the maximum_value with a different formula.